packages/realm/src/app-services/SyncSession.ts

```diff
@@ -262,6 +263,23 @@ const CONNECTION_LISTENERS = new Listeners<ConnectionNotificationCallback, Liste
 });
 
 export class SyncSession {
+  private static instances = new Set<binding.WeakRef<SyncSession>>();
```

Comment:
We have the `@internal` on the instances on `ProgressRealmPromise`. Did we decide which one we wanted to converge on?

Reply:
FWIW, there is less (but still not 0) benefit from keeping the `private static` members in the .d.ts vs the `private` instance members.